salt/cert/init.sls

```diff
@@ -2,7 +2,7 @@ include:
   - crypto
 
 {% set ip_addresses = [] -%}
-{% set extra_names = ["DNS: " + grains['fqdn']] -%}
+{% set extra_names = ["DNS: " + grains['caasp_fqdn'] ] -%}
```

Comment:
Many TLS implementations now ignore the certs CN when SANs are present, i.e. the CN should always be included as a SAN.